lib/X25519KeyAgreementKey2019.js

```diff
@@ -127,15 +127,23 @@ export class X25519KeyAgreementKey2019 extends LDVerifierKeyPair {
    * @returns {X25519KeyAgreementKey2019}
    */
   static fromEd25519VerificationKey2020({keyPair}) {
+    if(!keyPair.publicKeyMultibase) {
+      throw new Error('Source public key is required to convert.');
+    }
+
     const xKey = new X25519KeyAgreementKey2019({
       controller: keyPair.controller,
       publicKeyBase58: X25519KeyAgreementKey2019
-        .convertFromEdPublicKey(keyPair.publicKeyMultibase.substr(1))
+        .convertFromEdPublicKey({
+          publicKeyBase58: keyPair.publicKeyMultibase.substr(1)
```

Comment:
I see that blindly stripping the first character here is a carry-over.  But, it seems like it would be a good idea to test that the first character is a `z` before assuming it's a proper base58 encoded value.

Reply:
Ah good call.

Reply:
Added the check.